app/i18n/he/conf.php

```diff
@@ -92,7 +92,7 @@
 		'auto_remove_article' => 'Hide articles after reading',	//TODO - Translation
 		'confirm_enabled' => 'הצגת דו-שיח לאישור “סימון הכל כנקרא” ',
 		'display_articles_unfolded' => 'הצגת מאמרים בשלמותם כברירת מחדל',
-		'display_categories_unfolded' => 'הצגת קטגוריות מקופלות כברירת מחדל',
+		'display_categories_unfolded' => 'הצגת קטגוריות מקופלות כברירת מחדל',	//TODO
```

Comment:
Btw, this is a bit riskier because I don't really know Hebrew but it's fairly clear which work is folded vs unfolded/whole. A grammatical mistake might still be preferable to displaying the opposite of what's actually intended. Pinging @mat-mo.

```suggestion
		'display_categories_unfolded' => 'הצגת קטגוריות בשלמותם כברירת מחדל',	//TODO
```

Reply:
בשלמותם should be בשלמותן. Other than that, looks good.

Reply:
@mat-mo I corrected the suggestion in my comment, thanks!